net/instaweb/rewriter/redirect_on_size_limit_filter_test.cc

```diff
@@ -204,7 +215,7 @@ TEST_F(RedirectOnSizeLimitFilterTest, TestEscapingAndFlush) {
       "<table><tr><td>blah</td></tr></table></html>");
   html_parse()->FinishParse();
 
-  EXPECT_STREQ(kOutput, output_);
+  EXPECT_TRUE((kOutput == output_) || (kOutputAlt == output_)) << output_;
```

Comment:
There was a separate CL that tweaked the test, but I didn't think it was
important to backport.


On Tue, Sep 6, 2016 at 6:37 PM, hillsp <notifications@github.com> wrote:

> In net/instaweb/rewriter/redirect_on_size_limit_filter_test.cc
> <https://github.com/pagespeed/mod_pagespeed/pull/1387#discussion_r77731634>
> :
>
> > @@ -204,7 +215,7 @@ TEST_F(RedirectOnSizeLimitFilterTest, TestEscapingAndFlush) {
> >        "<table><tr><td>blah</td></tr></table></html>");
> >    html_parse()->FinishParse();
> >
> > -  EXPECT_STREQ(kOutput, output_);
> > +  EXPECT_TRUE((kOutput == output_) || (kOutputAlt == output_)) << output_;
>
> Wasn't there a newer version of this fix that doesn't require kOutputAlt?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pagespeed/mod_pagespeed/pull/1387/files/1bb708b41179647470813590ec15dedeb5ca98e9#r77731634>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADl1RMORmeE5mc6HqAmddIdGH5AAoWLJks5qnes1gaJpZM4J19oF>
> .
>